test/test_train_imagenet.py

```diff
@@ -50,15 +94,25 @@
     setattr(FLAGS, arg, value)
 
 
+def load_torchvision_model(model_name):
```

Comment:
We have a couple of places where we do `if inception`.
Maybe we can have something like this?
```Python
MODEL_PROPERTIES = {
  'inception': {
    img_dim: 299,
    model_fn: lambda: torchvision.models.inception_v3(aux_logits=False),
  },
  'DEFAULT': {
    img_dim: 224,
    model_fn: getattr(torchvision.models, FLAGS.model),
  }
}

def get_model_property(name):
  pdict = MODEL_PROPERTIES.get(FLAGS.model, None) or MODEL_PROPERTIES['DEFAULT']
  return pdict[name]
```